docs/content/meta/release-notes.md

```diff
@@ -42,25 +46,32 @@ Not only is this a much simpler model to understand, it is also faster and paved
 
 ## Other New Features
 
+* Every `Page` now has a `Kind` property. Since everything is a `Page` now, the `Kind` is used to differentiate different kinds of pages.
+  Possible values are `page`, `home`, `section`, `taxonomy`, `taxonomyTerm`, `RSS`, `sitemap`, `robotsTXT`, and `404`.
```

Comment:
This is true, but it may confuse some users. I deliberately kept all from RSS and right unexported, because they are only of internal interest (for now) -- these pages does not show up in any of the page collections and as such these kinds cannot be used for anything practical for end users. So I suggest just mentioning those in parens or something)